tools/bash/runfiles/runfiles_test.bash

```diff
@@ -153,6 +155,7 @@ EOF
   rm "$tmpdir/c/d" "$tmpdir/g h"
   [[ -z "$(rlocation a/b)" ]] || fail
   [[ -z "$(rlocation e/f)" ]] || fail
+  [[ "$(rlocation y)" == "$tmpdir/y" ]] || fail
```

Comment:
This line belongs above the `rm` line (155). Below the `rm` call, there should be a `rmdir` to delete `$tmpdir/y` and here should be an assertion to check that `rlocation` can no longer look up `y`.